### PR TITLE
[MAT-1256] Validate Libraries use supported FHIR 4.0.1.

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
Verify FHIR Libraries are using the supported FHIR version 4.0.1.
- All FHIR Library resources are expected to declare FHIR version being used.
- New check during parsiong for CQL 'using...' declaration to include 4.0.1.
- Raise exception on all other cases.

Skipping bundle loader composite tests.
- Bonnie-on-FHIR does not currently support composite measures.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: [MAT-1256](https://jira.cms.gov/browse/MAT-1256)
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/measure-loader/mat_measure_files.rb [hunk 1]
```diff
@@ -55,6 +55,7 @@ def self.parse_cql_elm(library)
           elm = JSON.parse(Base64.decode64(Base64.decode64(content['data']['value'])), max_nesting: 1000)
         when 'text/cql'
           cql = Base64.decode64(Base64.decode64(content['data']['value']))
+          raise MeasureLoadingInvalidPackageException.new("Library FHIR version does not match FHIR 4.0.1.") unless cql.to_s.downcase.include? "using FHIR version '4.0.1'".downcase
```
Comment (adongare) [inline lib/measure-loader/mat_measure_files.rb:58]: roby lints will complain about the use of double quotes if there is no string interpolation. single quote is prefered

Comment (adongare) [reply lib/measure-loader/mat_measure_files.rb:58]: Also, do we need to provide library name as well? or generic message 'One or more libraries FHIR version does not match FHIR 4.0.1' something like.

Comment (jkotanchik-SB) [reply lib/measure-loader/mat_measure_files.rb]: Adding a generic message is pretty straightforward and it would encourage the user to check all of their libraries. 

Alternatively, the logic could check all libraries, instead of raising an exception on the first non-compliance, and report the offenders to the user.

Comment (jkotanchik-SB) [reply lib/measure-loader/mat_measure_files.rb]: Added a generic message to the exception that will get displayed on the error modal in Bonnie.
